usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -30,6 +30,7 @@ The currently available endpoints are listed in the following table.
 | -------- | ------- | ----------- |
 |[/api/v2/agency/<TOPTIER_AGENCY_CODE\>/](/api/v2/agency/012/)|GET| Returns agency overview information for USAspending.gov's Agency Details page for agencies that have ever awarded. |
 |[/api/v2/agency/<TOPTIER_AGENCY_CODE\>/program_activity/count/](/api/v2/agency/012/program_activity/count/)|GET| Returns the count of unique Program Activity categories for the agency in a single fiscal year |
+|[/api/v2/agency/<TOPTIER_AGENCY_CODE\>/federal_account/count/](/api/v2/agency/012/federal_account/count/)|GET| Returns the count of unique Federal Accounts and Treasury Accounts for the agency in a single fiscal year |
```

Comment:
Are there non-unique federal accounts? I see the distinct, but I had assumed that was there to deal with the join statements in Django.

Reply:
I mean it's possible for an agency to have multiple treasury accounts that link back to the same federal account, so we want only the count of unique federal accounts with no duplicates. If the wording is unclear i can change it, but that's the idea.

Reply:
I think I'd be concerned as a user that there are two "equal" federal accounts floating around that I don't know about.

Reply:
What do you call "equal"?
We're counting the number of federal accounts. "One" = a federal account in the agency's File B in the given fiscal year. But a federal account isn't tied to a specific year or agency, right?